### PR TITLE
Ported few UI organzations tests

### LINKED ISSUE #6389: WebUI test coverage: Organization

### PR DESCRIPTION
**Pull request by abalakh**
Closes #6389 
Test results:
```python
pytest -v tests/foreman/ui_airgun/test_organization.py -k 'test_positive_delete_with_manifest_lces or test_positive_download_debug_cert_after_refresh'
============================== test session starts ===============================
platform darwin -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.22.0, services-1.2.1, mock-1.10.0, forked-0.2, env-0.6.2
collecting 12 items                                                              2018-11-27 11:36:22 - conftest - DEBUG - BZ deselect is disabled in settings

collected 12 items / 10 deselected

tests/foreman/ui_airgun/test_organization.py::test_positive_delete_with_manifest_lces PASSED [ 50%]
tests/foreman/ui_airgun/test_organization.py::test_positive_download_debug_cert_after_refresh SKIPPED [100%]

============== 1 passed, 1 skipped, 10 deselected in 354.04 seconds ==============
```

### CODE REVIEW

File: tests/foreman/ui/test_organization.py [hunk 1]
```diff
@@ -380,42 +306,6 @@ def test_negative_update(self):
                         common_locators['name_haserror'])
                     self.assertIsNotNone(error)
 
-    @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1321543)
-    @tier2
-    def test_positive_create_with_all_environment(self):
```
Comment (abalakh) [inline tests/foreman/ui/test_organization.py]: removed as BZ is closed with WONTFIX resolution

Comment (oshtaier) [reply tests/foreman/ui/test_organization.py]: did you check functionality?

File: tests/foreman/ui_airgun/test_organization.py [hunk 2]
```diff
@@ -293,3 +294,60 @@ def test_positive_update_location(session):
...
+    :id: 1fcd7cd1-8ba1-434f-b9fb-c4e920046eb4
+
+    :expectedresults: Scenario passed successfully
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    org = entities.Organization().create()
+    upload_manifest_locked(org.id)
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_organization.py]: Do you know that you need to take original manifest?

File: tests/foreman/ui_airgun/test_organization.py [hunk 3]
```diff
@@ -293,3 +294,60 @@ def test_positive_update_location(session):
...
+    with session:
+        session.organization.select(org.name)
+        for _ in range(3):
+            assert org.download_debug_certificate()
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_organization.py]: and now, probably you understand why it was necessary to remove manifest?

File: tests/foreman/ui/test_organization.py (hunk 1 above)
Comment (abalakh) [reply tests/foreman/ui/test_organization.py]: Yes, it's not fixed (like in BZ description).

File: tests/foreman/ui_airgun/test_organization.py (hunk 2 above)
Comment (abalakh) [reply tests/foreman/ui_airgun/test_organization.py]: Didn't notice that one manifest_upload was uploading smth different, unlike million other cases, thanks.

**Comment by abalakh** [comment]
@oshtaier addressed.
Test results (logs spam was so intensive i can't scroll till the beginning of run, attaching screenshot instead):
image

File: tests/foreman/ui/test_organization.py (hunk 1 above)
Comment (ldjebran) [reply tests/foreman/ui/test_organization.py]: I vote to not delete this test

**Comment by ldjebran** [comment]
+1 to not delete, put comment in doc string that bug 1321543  "CLOSED WONTFIX" and not ported for the moment.

**Pull request merged**